protos/flyteidl/admin/workflow_attributes.proto

```diff
@@ -46,14 +52,19 @@ message WorkflowAttributesGetResponse {
 
 message WorkflowAttributesDeleteRequest {
     // Unique project id which this set of attributes references.
+    // +required
     string project                 = 1;
 
     // Unique domain id which this set of attributes references.
+    // +required
     string domain                  = 2;
 
     // Workflow name which this set of attributes references.
+    // +required
     string workflow                = 3;
 
+    // Which type of matchable attributes to delete.
```

Comment:
lets put a reference to MatchableAttributes and in that proto explain what Matchable attributes really is

Reply:
sure, done